addons/account/static/src/js/tax_totals.js

```diff
@@ -63,9 +64,9 @@ class TaxGroupComponent extends Component {
      */
     _onChangeTaxValue() {
         this.setState("disable"); // Disable the input
-        let newValue = Number(this.inputTax.el.value); // Get the new value
+        let newValue;
         try {
-            newValue = formatFloat(newValue, { digits: this.props.currency.digits }); // Return a string rounded to currency precision
+            newValue = parseFloat(this.inputTax.el.value); // Get the new value
         } catch (_err) {
             this.props.invalidate();
             this.setState("edit");
```

Comment:
The behavior is weird when you input a non valid float
It will display the old value in the field, but it will tell you that the value is wrong when you try to save the record

Reply:
bug

Reply:
fixed